core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -100,6 +101,10 @@
 map(Routing, Payload) ->
     map_processor(Routing, Payload, get_binding_candidates(Routing)).
 
+-spec map(ne_binary(), payload(), kz_bindings()) -> map_results().
```

Comment:
But where is the call to the function? When I removed the export, `make xref` failed to complain about another module calling map/3.